auv_sim/auv_sim_bringup/launch/inc/start_world.launch.xml

```diff
@@ -16,8 +16,9 @@
     <arg name="verbose" value="true" />
   </include>
 
-  <node name="publish_world_models" pkg="uuv_assistants" type="publish_world_models.py"
-    output="screen">
+  <node if="$(eval arg('world_name') == '$(find auv_sim_description)/worlds/seabed.world')" 
+        name="publish_world_models" pkg="uuv_assistants" type="publish_world_models.py"
```

Comment:
This node is only relevant to the seabed world. Dont need it at times we run a world that is not the seabed world.